src/sfizz/Defaults.h

```diff
@@ -124,6 +124,41 @@ namespace Default
 	constexpr float rtDecay { 0.0f };
 	constexpr Range<float> rtDecayRange { 0.0f, 200.0f };
 
+    // Performance parameters: Filters
+    constexpr float filterCutoff { 0 };
+    constexpr float filterResonance { 0 };
+    constexpr int filterKeytrack { 0 };
+    constexpr uint8_t filterKeycenter { 60 };
+    constexpr int filterRandom { 60 };
+    constexpr int filterVeltrack { 0 };
+    constexpr int filterCutoffCC { 0 };
+    constexpr float filterResonanceCC { 0 };
+    constexpr Range<float> filterCutoffRange { 0.0f, 96000.0f };
```

Comment:
Filter's maximum at 96 kHz will not do.
It should be capped at Fs/2 maximum if possible, even a small bit lower.
With a minimum Fs at 44100 Hz, it seems not unreasonable to set it at 20kHz.

Reply:
In the end I think this parameter is more the real frequency anyway so 20 kHz is a much saner choice.